eppo/src/main/java/cloud/eppo/android/util/Utils.java

```diff
@@ -48,10 +82,33 @@ public static void validateNotEmptyOrNull(String input, String errorMessage) {
         }
     }
 
+    public static Date parseUtcISODateElement(JsonElement isoDateStringElement) {
+        if (isoDateStringElement == null || isoDateStringElement.isJsonNull()) {
+            return null;
+        }
+        String isoDateString = isoDateStringElement.getAsString();
+        Date result = null;
+        try {
+            result = isoUtcDateFormat.parse(isoDateString);
+        } catch (ParseException e) {
+            /* no-op; leave result null for now */
```

Comment:
If this catch scenario expected to happen during normal operation? Or would it be a symptom of something being broken/used incorrectly?

Reply:
This would be expected if parsing a base-64 encoded (obfuscated) date. I can expand the comment to indicate it's expected.